pages/conference/young-inspirers.vue

```diff
@@ -203,6 +232,128 @@ export default {
+            ],
+            speechInfos: [
+                {
+                    date: '23',
+                    month: 'July',
+                    speeches: [
+                        {
+                            tag: 'speech1',
+                            speechdate: 'July 23',
+                            speechtime: '14:00-14:40 (GMT+8)',
+                            title: 'Learning Python - A Journey',
+                            speaker: [
+                                {
+                                    name: 'Cheung Chun Lok Amos',
+                                    description: [
+                                        'I started my Python journey in Primary 4 when I joined Hour of Code Hong Kong. I was first introduced to programming and Python through the free Hour of Code lectures and workshops. The Hour of Code community provided me with the first resources I had to learn Python and STEM knowledge with, such as book recommendations and learning sites. Through the Internet, books, online documentations, learning sites and especially talks from peers and experts with experience in Python, I started to learn to make small projects and games with packages such as Tkinter and Turtle, which I’d share with others and get feedback from. Slowly, I began to develop a deep interest and passion for learning programming and for coding Python, a passion that I still have today. The more I learn about Python, the more I discover the infinite possibilities and projects I can create with it. Python is a popular language with a lot of potential for usage in machine learning and data science, and I hope to continue learning more and experimenting more on using Python in those fields and packages designed for those purposes, such as Tensorflow and Pandas, in the future.',
+                                    ],
+                                },
+                            ],
+                            full_description: [
+                                'In this talk, I wish to share my Python learning journey from the beginning, when I did not know how to code, and to inspire other young people who are starting out or wish to start. Free courses, workshops and online platforms, competitions all helped me out when I began my journey, and I also hope to share about what made me pursue a learning path in Python and in programming as well as my plans for my learning path, such as learning about machine learning and vision AI. Coding is an essential skill in the future as it is an important part of the digital transformation. It can possibly change the world by changing the way we live and complete tasks more efficiently. Thus, through this sharing, I wish to inspire young people like myself in the past who want to start but do not know where as well as offer tips on problem solving, advice on setting learning goals, and helpful information such as useful learning platforms for them so that they too can learn this essential skill.',
```

Comment:
```suggestion
                                'In this talk, I wish to share my Python learning journey from the beginning, when I did not know how to code, and to inspire other young people who are starting out or wish to start. Free courses, workshops, online platforms, and competitions all helped me out when I began my journey, and I also hope to share what made me pursue a learning path in Python and in programming as well as my plans for my learning path, such as learning about machine learning and vision AI. Coding is an essential skill in the future as it is an important part of the digital transformation. It can possibly change the world by changing the way we live and complete tasks more efficiently. Thus, through this sharing, I wish to inspire young people like myself in the past who want to start but do not know where as well as offer tips on problem-solving, advice on setting learning goals, and helpful information such as useful learning platforms for them so that they too can learn this essential skill.',
```